cmd/prepare.sh

```diff
@@ -1,8 +1,21 @@
 #!/bin/bash
 set -e;
 
+function check_docker_memory(){
+  DOCKER_TOTAL_MEMORY=`docker info | grep "Total Memory" | awk -F' ' '{print $3;}'`;
+  if [ -z "${DOCKER_TOTAL_MEMORY##*MiB}" ]; then
+    echo " Docker 'Total Memory' is measured in MiB, this is too low to run prepare polylines, please increase. See 'docker info' for full configuration."
+    exit 1
+  fi
+
+  if [ ${DOCKER_TOTAL_MEMORY/GiB/} \< 7.6 ]; then 
```

Comment:
Should this be `4.0` seems to conflict with the message below?

Reply:
Is that number based on something we published, I seem to recall it took more to do the whole planet?

Reply:
Also temped to merge this https://github.com/pelias/polylines/pull/248